packages/expo/package.json

```diff
@@ -71,7 +71,6 @@
     "@expo/config-plugins": "8.0.4",
     "@expo/metro-config": "0.18.4",
     "@expo/vector-icons": "^14.0.0",
-    "babel-preset-expo": "~11.0.6",
```

Comment:
Unfortunately, we can't do this. `babel-preset-expo` is currently only installed in users' projects through `expo.` If we remove this, projects will break because the preset won't be installed.

It's one of the implicit dependency chains I hope to get rid of. But for now, it has to stay in.

Reply:
Got it, I've added it again.